packages/babel-runtime/helpers/possibleConstructorReturn.ts

```diff
@@ -0,0 +1,9 @@
+var _typeof = require("./typeof.js")["default"];
+var assertThisInitialized = require("./assertThisInitialized.js");
+
+function _possibleConstructorReturn<T, U>(t: T, e: U): U | T {
```

Comment:
Is it possible to migrate to `export default`? Let's give it a shot and see if the `make test` passes.

Reply:
great!
on it

Reply:
should I also convert `require()` to use `esm import?`
@SukkaW

Reply:
The files in this directory should be automatically generated and we should not edit it manually.

Reply:
oh yeah you're right

there is another file called possibleConstructorReturn.js in packages/babel-helpers/src/helpers/possibleConstructorReturn.js

which has the same name as this one

should I also convert it to typescript?

Reply:
Yes, we should convert it and remove this.

Reply:
fixed and pushed